.gitignore

```diff
@@ -25,3 +25,9 @@ yarn-error.log*
 
 /.idea
 .vscode
+
+# package directories
+jspm_packages
+
+# Serverless directories
+.serverless
```

Comment:
It is customary to end each source file with a new line. Helps with cleaner diffs when adding stuff to the bottom, among other things.

Reply:
Fixed.